openmetadata-ui/src/main/resources/ui/playwright/support/entity/EntityClass.ts

```diff
@@ -21,23 +21,39 @@ import {
   assignGlossaryTerm,
   assignTag,
   assignTier,
+  downVote,
+  followEntity,
   removeGlossaryTerm,
   removeOwner,
   removeTag,
   removeTier,
+  unFollowEntity,
   updateDescription,
   updateOwner,
+  upVote,
+  validateFollowedEntityToWidget,
 } from '../../utils/entityUtils';
 import { Domain } from '../domain/Domain';
 import { GlossaryTerm } from '../glossary/GlossaryTerm';
+import { EntityType } from './Entity.interface';
 
 export class EntityClass {
   type: string;
+  endpoint: EntityType;
```

Comment:
@ShaileshParmar11 , we have `type` variable so maybe we can utilize that for entityType instead of creating another variable `endpoint`.

Reply:
The endpoint is the URL that we need for the request, and the type is Entity Type.

Reply:
From the type I can see that `type` is string and `endpoint` is entityType, so can we update the type accordingly?

Reply:
changes push